user-guide/Advanced_Functionality/DataMiner_Agents/Uninstalling_a_DataMiner_Agent.md

```diff
@@ -55,7 +55,7 @@ There are two possibilities to uninstall a DataMiner Agent:
 
    - *UnRegister DataMiner.bat*
 
-   - *UnRegister DLLs of DataMiner.bat*
+   - *UnRegister dll's of DataMiner.bat*
```

Comment:
This spelling is incorrect, but it is indeed the spelling used for this .bat file in the Tools folder. Is there any chance this could get corrected in some upcoming DataMiner version?

Reply:
Maybe this could be solved, but our team does not have enough expertise to assess the impact of such change. I assume team of @OnyxBit could look into it.